common/js/src/popup-window/in-popup-main-script.js

```diff
@@ -115,7 +117,11 @@ GSC.InPopupMainScript.setWindowHeightToFitContent = function() {
           typeof wholeContentHeight === 'number');
   goog.log.fine(
       logger, 'Resizing the window size to ' + wholeContentHeight + 'px');
-  chrome.app.window.current().innerBounds.height = wholeContentHeight;
+  if (GSC.Packaging.MODE == GSC.Packaging.Mode.APP) {
+    chrome.app.window.current().innerBounds.height = wholeContentHeight;
+  } else if (GSC.Packaging.MODE == GSC.Packaging.Mode.EXTENSION) {
+    chrome.windows.height = wholeContentHeight;
```

Comment:
Hmm, does it work? Seems odd - this, basically, creates a property called `height` in the `chrome.windows` namespace. I don't think that's how you set the opened window height.